server/src/app.js

```diff
@@ -1,34 +1,26 @@
-const express = require("express");
-const { join } = require("path");
-const compression = require("compression");
-const cookieParser = require("cookie-parser");
+const express = require('express');
+const { join } = require('path');
+const compression = require('compression');
+const cookieParser = require('cookie-parser');
 
-const router = require("./router");
+const router = require('./router');
 
 const app = express();
 
-app.disabled("x-powered-by");
+app.disabled('x-powered-by');
 
-app.set("port", process.env.PORT || 5000);
+app.set('port', process.env.PORT || 5000);
 
 const middlewares = [
   compression(),
   cookieParser(),
   express.json(),
-  express.urlencoded({ extended: false })
+  express.urlencoded({ extended: false }),
 ];
 
 app.use(middlewares);
 
-app.use(express.static(join(__dirname, "..", "..", "client", "public")));
+app.use(express.static(join(__dirname, '..', '..', 'client', 'public')));
```

Comment:
this should be something line this because we are a restful api, and only in production we send the index.html and the rest of the files
```suggestion
if (NODE_ENV==='production') {
  //  i think one more route for the index.html
  app.use(express.static(join(__dirname, '..', '..', 'client', 'build')));
}
```
it would be cool if you move it to router.js

Reply:
an app.get('/', send file thing)